src/components/UserActions/ClickedDocumentList.ts

```diff
@@ -90,13 +90,18 @@ export class ClickedDocumentList extends Component {
         super(element, ClickedDocumentList.ID, bindings);
```

Comment:
```suggestion
        super(element, ClickedDocumentList.ID, bindings) /* istanbul ignore next Istanbul issue with next*/;
```

Just so we got 💯 coverage (I know we don't need nor want to aim 💯 but we can, however, aim to that and improve the coverage by removing some erratic thing like this issue with Istanbul.

Reply:
k